import_menu_items.php

```diff
@@ -14,19 +14,19 @@
     $spreadsheetItems = $reader->load("./xlsx/Items.xlsx");
 
     $sheetDataItems = $spreadsheetItems->getActiveSheet()->toArray();
-    // print_r($sheetDataItems); 
     
 
     $categories =$category_model->getAllCategory();
     unset($sheetDataItems[0]);
-    // print_r($sheetDataItems[0]);
+    
     $keys = array_column($categories, 'category_name');
 
     foreach($sheetDataItems as $row){
-        // print_r($row);
         $index = array_search($row[0], $keys);
-        // print_r($categories[$index]["id"]);
-        // $category_model->getCategoryId($row[0]);
+
+        if(!$categories[$index]["id"]){
```

Comment:
This should be corrected as below
if(!isset($categories[$index])){
    throw new Exception("message");
}

Reply:
Correction done.